tests/tt_eager/tensors/test_async_tensor_apis.cpp

```diff
@@ -198,33 +198,34 @@ TEST_F(CommonFixture, TestAsyncRefCountManager) {
     device->set_worker_mode(WorkExecutorMode::SYNCHRONOUS);
 }
 
-TEST_F(CommonFixture, TestAsyncEltwiseBinaryAutoFormat) {
-    // Test usecase where both inputs and outputs are on host and autoformat is used
-    Device* device = this->devices_[0];
-    device->set_worker_mode(WorkExecutorMode::ASYNCHRONOUS);
-    AutoFormat::SetDefaultDevice(device);
+// #5044: Removed Autoformat from EltwiseBinary
```

Comment:
Delete this instead of just commenting out

Reply:
ok.